File: programming-essentials/pe-02/practice.js
const traineeName = "Hussein";
const age = 20;
let completedLessons = 5;
let isReady = false;
const trainerName = null;
const learningLanguage = "Python";

completedLessons = 6;

const status = isReady
    ? "Ready to continue"
    : "Still learning";

console.log(`Name: ${traineeName}`);
console.log(`Age: ${age}`);
console.log(`Completed lessons: ${completedLessons}`);
console.log(`Ready: ${isReady}`);
console.log(`Trainer: ${trainerName ?? "Not assigned"}`);
console.log(`Learning language: ${learningLanguage}`);
console.log(`Status: ${status}`);

File: programming-essentials/pe-03/practice.py
trainee_name = "Hussein"
age = 20
completed_lessons = 5
is_ready = False
trainer_name = None
learning_language = "Python"

completed_lessons = 6

if is_ready:
    status = "Ready to continue"
else:
    status = "Still learning"

print(f"Name: {trainee_name}")
print(f"Age: {age}")
print(f"Completed lessons: {completed_lessons}")
print(f"Ready: {is_ready}")
print(f"Trainer: {trainer_name if trainer_name else 'Not assigned'}")
print(f"Learning language: {learning_language}")
print(f"Status: {status}")